packages/lab/src/components/Wizard/Wizard.stories.tsx

```diff
@@ -11,7 +11,7 @@ import {
   HvTypography,
 } from "@hitachivantara/uikit-react-core";
 import mockText from "./mockData";
-import HvWizardContext from "./WizardContext";
+import { HvWizardContext } from "./WizardContext";
```

Comment:
Can you instead import everything from  `.`? All these pieces are part of `Wizard`'s API, so it's an extra validation for these types of issues, and that the sub-components are correctly exported to the end user.

```suggestion
import { HvWizardContext } from ".";
```

Reply:
Yep, you're right. I'll change it 👍